NERODevelopment/content/DetailDisplay.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.0
+
+Item {
+    id: detailDisplay
+    property string shutdownFlowTask: "Task"
+    property string status: "null"
+    property bool ok: true
+    property bool clear: false
+    property bool highlight: true
+    property bool displayModal: false
+
+    width: 200
+    height: 50
+
+    Row {
+        id: row
+        width: parent.width
+        height: parent.height
+        anchors.centerIn: parent
+
+        Rectangle {
+            id: taskDisplay
+            color: highlight ? "lightgrey" : "transparent"
```

Comment:
this needs to be a gradient and ideally animates/ pulses like the mock looks like